### PR TITLE
docs(ui5-task-zipper): fix onlyZip option

### PR DESCRIPTION
**Pull request by nicoschoenteich**
Shouldn't this say `false`? If set to `true`, it will only generate the zip.

### CODE REVIEW

**Comment by nicoschoenteich** [comment]
Updated the commit message, the check still fails though, as it wouldn't pick up the new message.

File: packages/ui5-task-zipper/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Default value: `<app-id.zip>`
 List of files to be included in the ZIP archive relative to the project root or Map of of files to be included in the ZIP archive relative to the project root and target path in the ZIP archive.
 
 - onlyZip: `true|false`
-Set this to `true` if you also want to generate the unzipped resources in the `dist` folder. Otherwise, it will only create the zipped archive.
+Set this to `false` if you also want to generate the unzipped resources in the `dist` folder. Otherwise, it will only create the zipped archive.
```
Comment (petermuessig) [inline packages/ui5-task-zipper/README.md:33]: I would even phrase this completely different:

Set this to `true` to omit the resources contained in the zip from the build result (typically in the `dist` folder). By default, the build result contains all resources and the zip.

**Review by petermuessig — CHANGES_REQUESTED** [review]

**Comment by petermuessig** [comment]
Can you please rebase to verify if the commit message issue still exists?

**Comment by petermuessig** [comment]
THX

**Pull request merged**